app/src/main/java/org/mozilla/fenix/wallpapers/WallpaperManager.kt

```diff
@@ -15,7 +16,7 @@ import org.mozilla.fenix.utils.Settings
  * Provides access to available wallpapers and manages their states.
  */
 class WallpaperManager(private val settings: Settings) {
-
+    val logger = Logger("WallpaperManager")
```

Comment:
Are these logs published somewhere or are they just for debugging?

Reply:
They go to the logcat console, it's just for debugging :)